source/quick-start.txt

```diff
@@ -10,3 +10,68 @@ Quick Start
    :depth: 2
    :class: singlecol
 
+.. include:: /includes/quick-start/overview.rst
+
+Set up Your Project
+-------------------
+
+Initialize with Go Mod
+~~~~~~~~~~~~~~~~~~~~~~
+
+Create a new directory and initialize your project with ``go mod``.
+
+.. code-block:: shell
+
+   mkdir go-quickstart
+   cd go-quickstart
+   go mod init go-quickstart
+
+.. _add-mongodb-dependency:
+
+Add MongoDB as a Dependency
```

Comment:
Are we assuming previous Go knowledge on the part of the user? If not, we should explicitly state that this will download the latest released version of the Go Driver and that the user can instead add a different version by doing `go get go.mongodb.org/mongo-driver@<version tag>`.

Reply:
ping @terakilobyte

Reply:
We aren't assuming expert previous knowledge, but a minimum requirement would be knowing how to install Go and having read about the different built in tools. 

Since this is a quick start, I'm hesitant to add information about installing specific versions. If they are new I worry that adding information about installing specific versions can lead them to research it for very little gain and adds to the perceived complexity.

Definitely open to a discussion about this!